core.js

```diff
@@ -994,6 +994,13 @@ class FileTypeParser {
 			};
 		}
 
+		if (this.check([0x41, 0x43])) {
```

Comment:
The [file signatures](https://www.garykessler.net/library/file_sigs.html) also shows that `41 43` is used by other file types (ACL & ACSD), so just testing on `41 43` and assume it's an AutoCAD file is a tricky assumption.

I propose to test on the following: `41 43 31 2E`, `41 43 32 2E`, `41 43 31 30`, where you can keep the initial test on `41 43`. After that you test on `31 2E`, `32 2E` & `31 30` at `offset=2`.

Please move the test to ~line 883, as the last test of the 4 byte signature group.

Reply:
The [DWG specification](https://www.opendesign.com/files/guestdownloads/OpenDesign_Specification_for_.dwg_files.pdf) explains the header in 3.2. The header is ASCII, and you can make use of that.

The `41 43` is actually `AC` (**A**uto**C**AD I guess). 
The 4 bytes after that give you the version. You can test of that if version is in [1000, 1050]. (The standard describes a range from [1012, 1032]. So there is some backward and future compatibility here.

That makes it a 6 bytes check.